plugins/importexport/medra/MedraExportPlugin.inc.php

```diff
@@ -102,11 +103,25 @@ function depositXML($objects, $context, $filename) {
 		assert(is_readable($filename));
 		$xml = file_get_contents($filename);
 		assert($xml !== false && !empty($xml));
-
+		
+		// Select the language
+		$language = 'en_US';
+		foreach($objects as $object) {
+		    $language = $object->getLocale();
+		}
+		$lang = 'eng';
+		if(substr($language, 0, 2) == 'it'){
```

Comment:
It should be possible to get the 3-letter codes from `registry/locales.xml`, using for example `PKPLocale::get3LetterFrom2LetterIsoLanguage`, rather than hard-coding a few language codes here and leaving the rest unsupported.

Reply:
done.

Reply:
I don't see the changes in the pull request yet; can you let me know when you've pushed them up?

Reply:
I pushed them up 5 minutes ago.

Reply:
Did you see the changes in the pull request? I made it 5 days ago. Let me know. Thanks